Forwarded: https://github.com/BuddiesOfBudgie/budgie-desktop/pull/124
Author: Campbell Jones <git@serebit.com>
Last-Update: 2022-05-16
Description: [PATCH4/5] Fix button icon disappearing after set_buttoncontent call

---
 src/daemon/screenshot.vala | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/daemon/screenshot.vala b/src/daemon/screenshot.vala
index c5f53343..4cf05dd9 100644
--- a/src/daemon/screenshot.vala
+++ b/src/daemon/screenshot.vala
@@ -1044,6 +1044,7 @@ namespace BudgieScr {
 			buttonimage.margin_left = 8;
 			buttonimage.margin_right = 8;
 			b.add(buttonimage);
+			b.show_all();
 		}
 
 		private Gdk.Pixbuf resize_pixbuf(Pixbuf pxb, int scale) {
-- 
2.34.1


